### PR TITLE
add auto case OCP-36414

### PR DESCRIPTION
**Pull request by zhaozhanqi**
@huiran0826 @anuragthehatter @rbbratta @weliang1 PTAL, thanks
logs;
http://pastebin.test.redhat.com/917450

### CODE REVIEW

**Comment by rbbratta** [comment]
looks like the logs pastebin expired.

**Comment by rbbratta** [comment]
I can't test, but it looks like we could simplify some step definitions

Would this work?

```diff
diff --git a/features/step_definitions/sriov.rb b/features/step_definitions/sriov.rb
index 217f0efd..03ab6d53 100644
--- a/features/step_definitions/sriov.rb
+++ b/features/step_definitions/sriov.rb
@@ -123,25 +123,15 @@ end
 Given /^I delete the #{QUOTED} sriov networkpolicy$/ do | cr_name |
   ensure_admin_tagged
   ensure_destructive_tagged
-  step %Q/I switch to cluster admin pseudo user/
-  step %Q/I use the "openshift-sriov-network-operator" project/
-  sriov_ns = "openshift-sriov-network-operator"
-  if sriov_network_node_policy(cr_name).exists?
-    @result = admin.cli_exec(:delete, object_type: 'sriovnetworknodepolicies', object_name_or_id: cr_name, n: sriov_ns)
-    raise "Unable to delete the sriov policy" unless @result[:success]
-  end
+  project("openshift-sriov-network-operator")
+  sriov_network_node_policy(cr_name).ensure_deleted(admin)
 end

 Given /^I delete the #{QUOTED} sriovnetwork$/ do | cr_name |
   ensure_admin_tagged
   ensure_destructive_tagged
-  step %Q/I switch to cluster admin pseudo user/
-  step %Q/I use the "openshift-sriov-network-operator" project/
-  sriov_ns = "openshift-sriov-network-operator"
-  if sriov_network(cr_name).exists?
-    @result = admin.cli_exec(:delete, object_type: 'sriovnetwork', object_name_or_id: cr_name, n: sriov_ns)
-    raise "Unable to delete the sriovnetwork" unless @result[:success]
-  end
+  project("openshift-sriov-network-operator")
+  sriov_network(cr_name).ensure_deleted(admin)
 end

 Given /^(SR-IOV resource injector|Admission webhook) is (enabled|disabled)$/ do | feature, status |
```

File: features/networking/sriov.feature [hunk 1]
```diff
@@ -761,10 +729,220 @@ Feature: Sriov related scenarios
...
+  @admin  
+  Scenario: VF number should be correct when two node have same PF name
+    Given the sriov operator is running well
+    When I run the :label admin command with:
+      | resource | node                                          |
+      | name     | dell-per740-13.rhts.eng.pek2.redhat.com       |
+      | key_val  | sriov=worker13                                |
+      | key_val  | feature.node.kubernetes.io/sriov-capable=true |
+      | overwrite| true                                          |
+    Then the step should succeed
```
Comment (liangxia) [inline features/networking/sriov.feature:808]: Given label "foo=bar" is added to the "xxx" node

**Comment by zhaozhanqi** [comment]
@anuragthehatter could you help add /lgtm if this is ok for you for merge. long time PR :)

**Pull request closed without merging**